http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppV2Routes.scala

```diff
@@ -19,51 +19,53 @@ import org.broadinstitute.dsde.workbench.model.UserInfo
 import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
 import org.http4s.Uri
 
-class AppV2Routes(kubernetesService: AppService[IO], userInfoDirectives: UserInfoDirectives)(implicit
+class AppV2Routes(kubernetesService: AppService[IO], userInfoDirectives: UserInfoDirectives, enabledUserDirectives: EnabledUserDirectives)(implicit
```

Comment:
Tons of whitespace changes in these files; reload with https://github.com/DataBiosphere/leonardo/pull/3218/files?w=1 to ignore